### ISSUE TITLE
Why are there no traces of archived notes?

### ISSUE THREAD

**Issue by nicolasfranck** [DMPRoadmap/roadmap#2817]
Please complete the following fields as applicable:

**What version of the DMPRoadmap code are you running? (e.g. v2.2.0)**

3.0.1

**Expected behaviour:**

One should be able to see a trace of archived notes in the edit form of a plan.
In the old DMPonline_v4 this was done by showing
"comment removed by <user>" with a line through it.
Now it simply discarded:

https://github.com/DMPRoadmap/roadmap/blob/master/app/views/notes/_list.html.erb#L3

Doesn't that result into a strange
conversation, where someone can actually redraw his/her
comment, and the next comment answers to something of
which there is no trace?

**Comment by briri** [comment]
yes, I suppose that is a possible scenario. It has not come up as an issue for us because I think this feature does not have a lot of adoption for us (We have around 50k plans in our system here and only around 2,500 comments in the notes table). Most of our users seem to manage their conversations offline via email.

@raycarrick-ed and @martaribeiro have either of you heard any complaints about this scenario in your instances?
